File: src/main/webapp/resources/modules/ticket/TicketController.js
function TicketController(TicketService, UserService) {
    var ticket = this;
    ticket.list = [];
    ticket.single = {};
    ticket.getTickets = function () {
        TicketService.getTickets().then(function (data) {
            ticket.list = data.reverse();
        });

        ticket.getUsers();
    };
    ticket.insertTicket = function() {
        TicketService.insertTicket(ticket.single).then(function (data) {
            ticket.list.unshift(data);
        });

        ticket.single = {};
    };
    ticket.deleteTicket = function (single) {
        TicketService.deleteTicketById(single.id).then(function (data) {
            ticket.list.splice(ticket.list.indexOf(single), 1)
        });
    };

    ticket.userList = [];
    ticket.getUsers = function () {
        UserService.getUsers().then(function (data) {
            ticket.userList = data;
        });
    };

    ticket.updateTicket = function (single, status) {
        single.status = status;
        TicketService.updateTicket(single);
    }
}

angular
    .module("staff")
    .controller("TicketController", TicketController);

File: src/main/java/com/bonvio/staff/controllers/ticketController.java
package com.bonvio.staff.controllers;

import com.bonvio.staff.models.Ticket;
import com.bonvio.staff.services.TicketService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.*;

import java.text.SimpleDateFormat;
import java.time.Instant;
import java.util.Date;
import java.util.List;

/**
 * Created by mil on 04.06.15.
 */

@Controller
@RequestMapping("ticket")
public class TicketController {

    @RequestMapping(method = RequestMethod.GET)
    public String main() {
        return "ticket";
    }

    @Autowired
    private TicketService ticketService;

    @RequestMapping(value = "/getTickets", method = RequestMethod.GET)
    @ResponseBody
    public List<Ticket> getAllUsers() {
        return ticketService.getAllTicket();
    }

    @RequestMapping(value = "/insertTicket", method = RequestMethod.POST)
    @ResponseBody
    public Ticket insertTicket(@RequestBody Ticket ticket) {
        //SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy.MM.dd G 'at' HH:mm:ss z");
        //System.out.println("date: " + dateFormat.format( new Date() ) );
        //ticket.setDateCreate(dateFormat.parse(dateFormat.format(new Date() )));
        ticket.setDateCreate(new Date());
        ticket.setStatus(1);
        //ticket.setDateCreate(Date.from(Instant.now()));
        ticketService.insertTicket(ticket);
        return ticket;
    }

    @RequestMapping(value =  "/deleteTicketById/{id}", method = RequestMethod.DELETE)
    @ResponseBody
    public Integer deleteTicketById(@PathVariable("id") Integer id) {
        ticketService.deleteTicketById(id);
        return id;
    }

    @RequestMapping(value = "/updateTicket", method = RequestMethod.PUT)
    @ResponseBody
    public Ticket updateTicket(@RequestBody Ticket ticket) {
        ticketService.updateTicket(ticket);
        return ticket;
    }

}
